### ISSUE TITLE
Wallpaper only loading on one screen/重启之后，两个屏幕只有一个生效，另外一个变成了原来的桌面

### ISSUE THREAD

**Issue by jingouwangzi** [rocksdanister/lively#170]
我有两个屏幕，两个屏幕分别设置了动态桌面，但是当电脑重启之后，只有一个屏幕的动态桌面还在，另外一个变成了原来的桌面。
I have two screens, and the two screens have dynamic desktops respectively set up, but when the computer restarts, only the dynamic desktop of one screen is still there, and the other one becomes the original desktop.（Translated by Google）

**Comment by jingouwangzi** [comment]
操作系统：win10 2004，今天升级到20H2，都是一样的问题。
Operating system: win10 2004, upgraded to 20H2 today, all have the same problem.

**Comment by rocksdanister** [comment]
Does this happen always? or sometimes?

**Comment by jingouwangzi** [comment]
昨天和前天出现过，今天又恢复正常了
Happened yesterday and the day before yesterday, and returned to normal today

**Comment by rocksdanister** [comment]
Share the log file(s) from Local Appdata folder` C:\Users\<UserName>\AppData\Local\Lively Wallpaper\logs`

**Comment by jingouwangzi** [comment]
[20201021_175826.txt](https://github.com/rocksdanister/lively/files/5427239/20201021_175826.txt)
[20201022_080105.txt](https://github.com/rocksdanister/lively/files/5427240/20201022_080105.txt)
[20201022_144639.txt](https://github.com/rocksdanister/lively/files/5427241/20201022_144639.txt)
[20201023_080106.txt](https://github.com/rocksdanister/lively/files/5427242/20201023_080106.txt)

**Comment by rocksdanister** [comment]
I took a look at the logs and and could not find anything wrong, I think the log file with the problem information is gone (Lively only keep last 5 copies and deletes the old log files.)

Next time this happens, close lively immediately from system-tray and share all the logfiles.

**Comment by jingouwangzi** [comment]
> I took a look at the logs and and could not find anything wrong, I think the log file with the problem information is gone (Lively only keep last 5 copies and deletes the old log files.)
> 
> Next time this happens, close lively immediately from system-tray and share all the logfiles.


OK. Thank you.

**Comment by jingouwangzi** [comment]
[20201026_080824.txt](https://github.com/rocksdanister/lively/files/5462130/20201026_080824.txt)
[20201027_080104.txt](https://github.com/rocksdanister/lively/files/5462132/20201027_080104.txt)
[20201028_080109.txt](https://github.com/rocksdanister/lively/files/5462133/20201028_080109.txt)
[20201028_080954.txt](https://github.com/rocksdanister/lively/files/5462134/20201028_080954.txt)
[20201029_141004.txt](https://github.com/rocksdanister/lively/files/5462136/20201029_141004.txt)
[20201030_080108.txt](https://github.com/rocksdanister/lively/files/5462137/20201030_080108.txt)
Happened again

**Comment by rocksdanister** [comment]
```2020-10-27 11:25:34.3258|INFO|livelywpf.SetupDesktop|System parameters changed: Screen Connected/Disconnected
2020-10-27 11:25:34.3427|INFO|livelywpf.SetupDesktop|System parameters changed: Disconnected Screen -> \\.\DISPLAY1 {X=1920,Y=0,Width=1920,Height=1080}
2020-10-27 11:25:35.4349|INFO|livelywpf.SetupDesktop|System parameters changed: Screen Connected/Disconnected
2020-10-27 11:25:36.0781|INFO|livelywpf.SetupDesktop|System parameters changed: Screen Connected/Disconnected
2020-10-27 11:25:38.5786|INFO|livelywpf.SetupDesktop|System parameters changed: Screen Connected/Disconnected
```

`Disconnected Screen -> \\.\DISPLAY1 {X=1920,Y=0,Width=1920,Height=1080}`
Windows display settings change event is firing and Lively believes your right screen is disconnected therefore closing the wallpaper on that screen.

Possible causes for the event to fire:
1. Is your display connector cable loose?
2. Do you have any third party software that interact with display?

**Comment by jingouwangzi** [comment]
可能因为我的显示器会关掉，早上开机的时候，显示器是关闭状态，所以软件检测不到？
Maybe because my monitor turns off at night, when I turn on the computer in the morning, the monitor is off, so the software can't detect it?

**Comment by rocksdanister** [comment]
That should not be an issue, is it using DisplayPort connector?

**Comment by jingouwangzi** [comment]
a DP and a HDMI

**Comment by rocksdanister** [comment]
So the screen on the right is using DisplayPort?
DisplayPort has this plug and play feature and some monitor don't implement it properly, that maybe it.
Like the monitor will actually disconnect from windows when going to sleep/standby.

**Comment by jingouwangzi** [comment]
我忘记了，它在办公室，可能是的。
I forget, it's in the office, maybe it is.

**Comment by rocksdanister** [comment]
Fixed with latest update v1.1.9.4
